### PR TITLE
Fixes for selinux

### PR DESCRIPTION
**Pull request by sblatt**
#527

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -69,7 +73,42 @@
     purge   => $purge,
     recurse => $recurse,
   }
+ 
+  if $collectd_selinux and $::selinux == 'true' {
```
Comment (vinzent) [inline manifests/config.pp]: ::selinux is a boolean and I'd expect comparing to a string will fail on Puppet4.

File: manifests/config.pp [hunk 2]
```diff
@@ -69,7 +73,42 @@
...
+    selboolean { 'collectd_tcp_network_connect':
+      persistent => true,
+      provider   => getsetsebool,
+      name       => collectd_tcp_network_connect,
+      value      => 'on',
+    }
+    file { 'collectd_udev.pp':
```
Comment (vinzent) [inline manifests/config.pp]: A binary is not portable across distros and not even across distro minor versions. A module compiled on EL 7.3 doest necessarily work on EL7.1 and fore sure wont work on EL6 or on Ubuntu.

**Comment by sblatt** [comment]
Thynk you for your feedback. We updated to Puppet 4, so I can test that. I will also look into the binary selinux file.

**Comment by oranenj** [comment]
I'm not sure if this belongs in the collectd module at all... At most it should be a completely optional manifest, and I'd not even add a dependency on puppet-selinux to metadata.json (when can we get soft dependencies?:()

Also, maybe $manage_selinux instead of $collectd_selinux?

**Comment by vinzent** [comment]
I'm closing this because of no feedback of author since january.

**Pull request closed without merging**